pkg/band/rpc.go

```diff
@@ -58,3 +60,141 @@ func GetPhyVersions(ctx context.Context, req *ttnpb.GetPhyVersionsRequest) (*ttn
 	}
 	return res, nil
 }
+
+// ListBands returns the list of supported bands.
+func ListBands(ctx context.Context, req *ttnpb.ListBandsRequest) (*ttnpb.ListBandsResponse, error) {
+	filteredVersions := make(map[string]map[ttnpb.PHYVersion]Band)
+
+	if req.BandId != "" {
+		versions, ok := All[req.BandId]
+		if !ok {
+			return nil, errBandNotFound.WithAttributes(req.BandId, req.PhyVersion.String())
```

Comment:
The `WithAttributes` expects key value pairings, so you want to use actually `("id", req.BandId, "version", req.PhyVersion)` here.